File: ManagemateAPI/ManagemateAPI/Management/M_Authorized_Worker/Input_Objects/Add_Authorized_Worker_Data.cs
using ManagemateAPI.Management.M_Session.Input_Objects;

namespace ManagemateAPI.Management.M_Authorized_Worker.Input_Objects
{
    public class Add_Authorized_Worker_Data
    {
        public Session_Data session { get; set; }
        public long client_id_FK { get; set; }
        public string name { get; set; }
        public string surname { get; set; }
        public string phone_number { get; set; }
        public string email { get; set; }
        public bool contact { get; set; }
        public bool collection { get; set; }
        public string comment { get; set; }
    }
}

File: Managemate/ManagemateAngular/src/app/shared/interfaces/API_Input_Models/Authorized_Worker_Models/Add_Authorized_Worker_Data.ts
import {Session_Data} from "../../API_Other_Models/Session_Models/SessionData";

export interface Add_Authorized_Worker_Data{
    session?: Session_Data;
    
    client_id_FK: number;
    name: string;
    surname: string;
    phone_number: string;
    email: string;
    contact: boolean;
    collection: boolean;
    comment: string;
}